proto/tendermint/state/types.proto

```diff
@@ -42,37 +42,37 @@ message State {
 
   // immutable
   string chain_id       = 2 [(gogoproto.customname) = "ChainID"];
-  int64  initial_height = 3;
+  int64  initial_height = 14;
 
   // LastBlockHeight=0 at genesis (ie. block(H=0) does not exist)
-  int64                    last_block_height = 4;
-  tendermint.types.BlockID last_block_id     = 5
+  int64                    last_block_height = 3;
+  tendermint.types.BlockID last_block_id     = 4
       [(gogoproto.nullable) = false, (gogoproto.customname) = "LastBlockID"];
-  google.protobuf.Timestamp last_block_time = 6
+  google.protobuf.Timestamp last_block_time = 5
       [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
 
-  tendermint.types.ChainLock last_chain_lock = 7 [(gogoproto.nullable) = false];
-  tendermint.types.ChainLock next_chain_lock = 8 [(gogoproto.nullable) = false];
+  tendermint.types.CoreChainLock last_core_chain_lock = 100 [(gogoproto.nullable) = false];
```

Comment:
This will lead to our system not being backward compatible if any value ever comes before 100. This should not be done.